test-packages/integration-tests/test/test-data/single-test-entity-response-v4.ts

```diff
@@ -0,0 +1,62 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
```

Comment:
Also create a ticket that we remember to setup a real E2E test with a CAP service or one of the two v4 services.

Reply:
It's on the radar, but you are right, a ticket is good for tracking.
-> https://github.wdf.sap.corp/MA/scp-rocket/issues/1258